sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobMetrics.java

```diff
@@ -97,9 +97,9 @@ public Boolean isIncludeAPIs() {
      * summary statistics for called API operations.
      *
      * @param includeAPIs the includeAPIs value to set.
-     * @return the Metrics object itself.
+     * @return the BlobMetrics object itself.
      */
-    public Metrics setIncludeAPIs(Boolean includeAPIs) {
+    public BlobMetrics setIncludeAPIs(Boolean includeAPIs) {
```

Comment:
Should this be setIncludeApis? And same with the getter?

Reply:
Will document this issue, this will require another Swagger transform.